src/Core/Geographic/GeoCoordinate.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GeoCoordinate
+ * Description: Coordonnées cartographiques
+ */
+/**
+ *
+ * @param {type} defaultValue
+ * @returns {CoordCarto_L9.GeoCoordinate}
+ */
+import defaultValue from 'Core/defaultValue';
+import mE from 'Core/Math/MathExtented';
+
+/* eslint-disable */
+var UNIT_RADIAN = 0;
+var UNIT_DEGREE = 1;
+var UNIT_METER = 2;
+
+var COORD_LONG = 0;
+var COORD_LAT = 1;
+var COORD_ALT = 2;
+/* eslint-enable */
+
+function GeoCoordinate(longitude, latitude, altitude,unit) {
+
+
+	this.angleUnit = defaultValue(unit,UNIT_RADIAN);
+
+	this.coordinate = [];
+
+	this.coordinate[this.angleUnit] = [];
+	this.vAltitude = [];
+
+	this.coordinate[this.angleUnit][COORD_LONG] = defaultValue(longitude, 0);
+	this.coordinate[this.angleUnit][COORD_LAT] = defaultValue(latitude, 0);
+
+	this.vAltitude[UNIT_METER] = defaultValue(altitude, 0);
+
+}
+
+GeoCoordinate.UNIT_DEGREE = UNIT_DEGREE;
+
+GeoCoordinate.prototype.constructor = GeoCoordinate;
+
+GeoCoordinate.prototype.longitude = function() {
+
+	return this.coordinate[this.angleUnit][COORD_LONG];
+};
+
+var _setCoordinateValue = function(angleUnit,coordinate,id,value)
```

Comment:
Any advantages over the `function _setCoordinateValue(angleUnit,coordinate,id,value)` syntax?